tests/test.h

```diff
@@ -75,34 +77,30 @@ inline std::string read_file_into_string(const std::string& filename)
     return read_file_into_string(f);
 }
 
-inline predicate_result is_same_as_file(const std::string& data, const std::string& filename)
-
+inline bool is_same_as_file(const std::string& data, const std::string& filename)
 {
     const std::string filedata = read_file_into_string(filename);
     if ( data == filedata )
         return true;
 
-    predicate_result res(false);
-    res.message() << "Expected output:\n";
-    res.message() << filedata;
-    res.message() << "\nActual output:\n";
-    res.message() << data;
-    return res;
+    FAIL_CHECK("Expected output:\n" << filedata << "\nActual output:\n" << data);
```

Comment:
@thesiv Just a question: is there any other reason to do this here rather than just `CHECK( data == filedata )` than that it was done like this in the old version?

Reply:
No other reason, just copy from previous version...